backend/src/api/routes/course/index.ts

```diff
@@ -110,6 +110,17 @@ router.post("/:courseId/enroll", fetchCourseMiddleware, async (req, res) => {
         })),
         skipDuplicates: true
     });
+    await prisma.role.deleteMany({
```

Comment:
maybe this could be a new route? "enroll" suggests append and not replace course list

Reply:
Mmm I see, yeah I'll do that